### ISSUE TITLE
Reviving megafauna corpse = easy loot and kill count

### ISSUE THREAD

**Issue by Namwty** [Skyrat-SS13/Skyrat-TG#25267]
## Reproduction:

You are able to do surgery on megafauna, which is clearly a overlook, as it seems it was designed for basic/simple mobs like reviving Ian.

So issue is, you can do revival on megafauna that leaves a corpse, mostly problematic with Ash Drake and The Marked One

1.
You can farm boxes from said bosses, by just doing surgery ( as it tends to revive said mob at like 1 hp anyways, unless you do tend wounds ). All it takes is for you to kill said megafauna once and some surgery tools, then you can start farming loot. I think Marked One also leaves a corpse, so ya would be able to do same, duplicating reward.

In short, someone can just spend 5 minutes reviving same drake and getting like 10 crate rewards, without fighting.

2.
Issue that some people care most likely, you get credited for those revival kills, meaning tumor that is achievements get's effected ( specifically kill count ) can be farmed with barely any work.

3.
I'll abstain from explaining how, but if you fuck around, ya can drag a corpse of a Ash drake to like station and revive it there to wreck havoc and probs other fun ways to break stuff. Generally I don't think staff want to babysit and keep eye if miner is abusing it or not ( bet it on low pop or high )

TL;DR
You can revive megafauna that leaves a corpse with 1~hp, meaning easy kill count for achievements and infinite loot from said bosses.

Now I don't know if it's TG related oversight, I'm not sure, nor I could have tested it there, so writing this issue here.
